### ISSUE TITLE
modern key signatures

### ISSUE THREAD

**Issue by rettinghaus** [pfefferniels/probstuecke-digital#76]
In some pieces the displayed key signature disagrees with the tonality from a modern perspective. 
On modernizing the clefs, shouldn't we also modernize the key signatures?

**Comment by pfefferniels** [comment]
Yes, I agree. Probably it would good to keep both options separate, since modern editions do often modernize clefs, but not the key signatures, as far as I know …

**Comment by rettinghaus** [comment]
I thought about that and I think it would be ok to combine both. Editions I know do modernize clefs and keys. From a practical point of view I think nobody will complain …
